### ISSUE TITLE
False positive for Hash values omission

### ISSUE THREAD

**Issue by AlexWayfer** [AtomLinter/linter-ruby#255]
![image](https://user-images.githubusercontent.com/6510020/188025736-72750710-72ba-4125-947e-bdb19c0bbf1e.png)

```
> ruby -c config/puma.rb
Syntax OK

> ruby -v
ruby 3.1.2p20 (2022-04-12 revision 4491bb740a) [x86_64-linux]
```
